diff --git a/CMakeLists.txt b/CMakeLists.txt
index 45c08ab..ff43fb3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,6 @@ find_package(ROOT REQUIRED)
 find_package( dunepdlegacy REQUIRED EXPORT )
 find_package( dunecore REQUIRED EXPORT ) 
 find_package( dunereco REQUIRED EXPORT ) 
-find_package( duneopdet REQUIRED EXPORT ) 
 find_package( duneanaobj REQUIRED EXPORT ) 
 find_package( larana REQUIRED EXPORT )
 find_package( larcoreobj REQUIRED EXPORT ) 
